src/notation/internal/notationconfiguration.cpp

```diff
@@ -888,3 +892,31 @@ void NotationConfiguration::setStyleFileImportPath(const io::path_t& path)
 {
     settings()->setSharedValue(STYLE_FILE_IMPORT_PATH_KEY, Val(path.toStdString()));
 }
+
+int NotationConfiguration::lastPageRow() const
+{
+    return m_lastPageRow;
+}
+
+void NotationConfiguration::setLastPageRow(int value)
+{
+    m_lastPageRow = value;
+}
+
+int NotationConfiguration::lastSubPageRow() const
+{
+    return m_lastSubPageRow;
+}
+
+void NotationConfiguration::setLastSubPageRow(int value)
+{
+    m_lastSubPageRow = value;
+}
+
+void NotationConfiguration::resetStyleIndex()
+{
+    if (context()->currentProject() == nullptr) {
```

Comment:
I don't think we need this `if` condition. As far as I can see, it doesn't have much effect, and if there will ever be a way to change from one project directly to another, i.e. skipping the intermediate state of having no open project, then the reset should happen too.